search-service/src/main/resources/application.properties

```diff
@@ -19,13 +19,17 @@ spring.security.oauth2.resourceserver.jwt.jwk-set-uri = https://sso.eglobalmark.
 
 spring.cloud.stream.kafka.binder.brokers = localhost:29092
 spring.kafka.bootstrap-servers = localhost:29092
+# To ensure we get all past messages when dynamically joining a new topic based on our "cim.entities.*" pattern
+# By default, new matching topics are checked every 5 minutes but it can be configured
```

Comment:
Could you explain ? is it a new spring kafka feature ?

Reply:
Explain the explanation? 🤔 What is not clear in it?

And, no, it's not new, we already have the same setting in subscription service. I spotted it during the configuration review following the upgrade.

Reply:
Sorry I don't know very much Kafka, how old the past messages could be ?